1_fetch/src/get_nwis_data.R

```diff
@@ -0,0 +1,79 @@
+
+get_nwis_daily_data<-function(site_num,outdir,parameterCd,startDate, endDate){
+   ##read daily NWIS data, and save to csv file.
+  data_out<-readNWISdv(site_num, parameterCd, startDate, endDate)
+  
+  filepath<-file.path(outdir,paste0(site_num, "_dv.csv"))
+  write_csv(data_out, file=filepath)
+  return(filepath)
+  
+}
+
+get_nwis_peak_data<-function(site_num,outdir,startDate, endDate){
+  ##read NWIS peak data, and save to csv file.
+  data_out<-readNWISpeak(site_num,  startDate, endDate)
+  
+  ##Removing any NA dates or peak_va values - dates are returned as NA if day of occurance is not known
+  ## and peak_va is sometimes returned as NA if the gage height is known but not the discharge.
+  data_out<-data_out[which(!is.na(data_out$peak_dt)),]
+  data_out<-data_out[which(!is.na(data_out$peak_va)),]
+  return(data_out)
+  
+}
+
+
+screen_daily_data<-function(filename,yearType){
```

Comment:
Could add a few lines to remove all provisional data. I'd think that stopping in 2020 will already get most of those, though.

Reply:
OK, have added a couple lines in the screening function to remove any provisional data prior to checking for complete years.  If we decide to remove estimated data, I can do that here as well.